controllers/gce/instances/fakes.go

```diff
@@ -151,20 +150,19 @@ func (f *FakeInstanceGroups) RemoveInstancesFromInstanceGroup(name, zone string,
 
 func (f *FakeInstanceGroups) SetNamedPortsOfInstanceGroup(igName, zone string, namedPorts []*compute.NamedPort) error {
 	found := false
-	for _, ig := range f.instanceGroups {
-		if ig.Name == igName && ig.Zone == zone {
+	var ig *compute.InstanceGroup
```

Comment:
Can remove the above found variable and use `ig == nil` for the "failed to find..." error.

Reply:
Done